pipeline/terraform/main.tf

```diff
@@ -38,3 +38,44 @@ module "catalogue_pipeline_20190916" {
   vhs_sierra_sourcedata_bucket_name = "${local.vhs_sierra_sourcedata_bucket_name}"
   vhs_sierra_sourcedata_table_name  = "${local.vhs_sierra_sourcedata_table_name}"
 }
+
+module "catalogue_pipeline_20191007" {
+  source = "stack"
+
+  namespace = "catalogue-20191007"
+
+  release_label = "prod"
+
+  account_id      = "${data.aws_caller_identity.current.account_id}"
+  aws_region      = "${local.aws_region}"
+  vpc_id          = "${local.vpc_id}"
+  subnets         = ["${local.private_subnets}"]
+  private_subnets = ["${local.private_subnets}"]
+
+  dlq_alarm_arn = "${local.dlq_alarm_arn}"
+
+  # Transformer config
+  #
+  # If this pipeline is meant to be reindexed, remember to uncomment the
+  # reindexer topic names.
+
+  sierra_adapter_topic_names = [
+    "${local.sierra_reindexer_topic_name}",
+    "${local.sierra_merged_bibs_topic_name}",
+
+    "${local.sierra_merged_items_topic_name}",
+  ]
+  miro_adapter_topic_names = [
+    "${local.miro_reindexer_topic_name}",
+    "${local.miro_updates_topic_name}",
+  ]
+  # Elasticsearch
+  es_works_index = "v2-20191007"
```

Comment:
We sometimes add a few words saying why we created the new index...  not sure if that's actually useful though.

Reply:
I don't find it particularly useful (might be useful immediately after creating a new index, but generally not important why the index was originally created a few weeks down the line). Have already run a full reindex on this, so don't think it is worth changing now, but happy to add a suffix on future ones.

Reply:
💯 Was just a note which made me realise I was following blindly.